api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -755,4 +761,172 @@ public void copyFile_shouldCopyInputstreamToOutputstreamAndCloseTheOutputstream(
+	}
+
+	/**
+	 * @see OpenmrsUtil#isInNormalNumericRange(Float, ConceptNumeric)
+	 */
+	@Test
+	public void isInNormalNumericRange_shouldNotAllowNullValuesAsLowerAndHigherNormal() {
+		
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiNormal(null);
+		boolean isInRange = OpenmrsUtil.isInNormalNumericRange(5.67f, concept);
+
+		assertFalse(isInRange);
+	}
+	
+	@Test
+	public void isInNormalNumericRange_shouldPassIfValueWithinRange() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiNormal(10.34d);
+		concept.setLowNormal(3.67d);
+		boolean isInRange = OpenmrsUtil.isInNormalNumericRange(5.67f, concept);
+
+		assertTrue(isInRange);
+	}
+
+	/**
+	 * @see OpenmrsUtil#isInCriticalNumericRange(Float, ConceptNumeric)
+	 */
+	@Test
+	public void isInCriticalNumericRange_shouldNotAllowNullValuesAsLowerAndHigherCritical() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiCritical(null);
+		boolean isInRange = OpenmrsUtil.isInCriticalNumericRange(5.67f, concept);
+
+		assertFalse(isInRange);
+	}
+
+	@Test
+	public void isInCriticalNumericRange_shouldPassIfValueWithinRange() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiCritical(10.34d);
+		concept.setLowCritical(3.67d);
+		boolean isInRange = OpenmrsUtil.isInCriticalNumericRange(5.67f, concept);
+
+		assertTrue(isInRange);
+	}
+
+	/**
+	 * @see OpenmrsUtil#isInAbsoluteNumericRange(Float, ConceptNumeric)
+	 */
+	@Test
+	public void isInAbsoluteNumericRange_shouldNotAllowNullValuesAsLowerAndHigherAbsolute() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiAbsolute(null);
+		boolean isInRange = OpenmrsUtil.isInAbsoluteNumericRange(5.67f, concept);
+
+		assertFalse(isInRange);
+	}
+
+	@Test
+	public void isInAbsoluteNumericRange_shouldPassIfValueWithinRange() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiAbsolute(10.34d);
+		concept.setLowAbsolute(3.67d);
+		boolean isInRange = OpenmrsUtil.isInAbsoluteNumericRange(5.67f, concept);
+
+		assertTrue(isInRange);
+	}
+
+	/**
+	 * @see OpenmrsUtil#isValidNumericValue(Float, ConceptNumeric)
+	 */
+	@Test
+	public void isValidNumericValue_shouldPassIfHigherOrLowerAbsoluteIsNull() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiAbsolute(null);
+		boolean isValid = OpenmrsUtil.isValidNumericValue(5.67f, concept);
+
+		assertTrue(isValid);
+	}
+
+	@Test
+	public void isValidNumericValue_shouldPassIfValueWithinAbsoluteRange() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiAbsolute(10.34d);
+		concept.setLowAbsolute(3.67d);
+		boolean isValid = OpenmrsUtil.isValidNumericValue(5.67f, concept);
+
+		assertTrue(isValid);
+	}
+
+	/**
+	 * @see OpenmrsUtil#getRuntimePropertiesFilePathName(String)
+	 */
+	@Test
+	public void getRuntimePropertiesFilePathName_shouldPassIfPropertyFileNotExist() {
+		
+		String pathName = OpenmrsUtil.getRuntimePropertiesFilePathName("app-openmrs");
```

Comment:
please inline variable. same reason as above. if creating a variable for example for choosing a meaningful name helps reading than create one. but here its just a one-liner.